src/Microsoft.DotNet.Wpf/src/PresentationCore/ModuleInitializer.cs

```diff
@@ -26,17 +26,19 @@ public static void Initialize()
     private static void IsProcessDpiAware()
     {
         // By default, Application is DPIAware.
-        Assembly assemblyApp = Assembly.GetEntryAssembly();
+        bool dpiAware = true;
 
         // Check if the Application has explicitly set DisableDpiAwareness attribute.
+        Assembly assemblyApp = Assembly.GetEntryAssembly();
         if (assemblyApp != null && Attribute.IsDefined(assemblyApp, typeof(System.Windows.Media.DisableDpiAwarenessAttribute)))
         {
-            // DpiAware composition is enabled for this application.
```

Comment:
Could you keep the comments at the same place as the previous C++ code? You can use this code for reference:
https://github.com/dotnet/wpf/blob/560f243910f407df33fd5cdf06e2a857684f4bc7/src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/main.cpp#L184-L205

Reply:
I changed it it to match the original C++ code more closely. I renamed `bDisableDpiAware` to `disableDpiAware`, so that it matches typical C# conventions.